### ISSUE TITLE
[BUG] Mix at 0%;0% on power up (Marlin 2.0.5.2)

### ISSUE THREAD

**Issue by Exalik** [MarlinFirmware/Marlin#17397]
### Bug Description

At power up, the mix is at 0%;0%

If a print is launching without change the mix ratio, the printer freeze just after the bed leveling.
[Marlin 2.0.5.2 config.zip]

### My Configurations

(https://github.com/MarlinFirmware/Marlin/files/4430894/Marlin.2.0.5.2.config.zip)

### Steps to Reproduce

1. Power up the printer
2. Look the Auto Home and the nozzle to go up for about 5mm
3. Look the beautiful screen... freezed (lol)

**Expected behavior:** The print should start and the screen should not freeze

**Actual behavior:** The print does not start and the screen is frozen

#### Additional Information

Geeetech A10M (GT2560 v3.1) with TMC2208 and 3DTouch

**Comment by pitercal99** [comment]
I'm also having the same problem
I tried to save on EEPROM but V-Tools settings are not included into the EEPROM data

### EDIT
I fix the problem editing the source code **_mixing.cpp_** (_"Marlin\src\feature\mixing.cpp"_)

The void **_Mixer::reset_vtools()_** function (line 98) is responsible to set the default mixing percentage on v-tools, I commented the original function and forced the value manually
```
void Mixer::reset_vtools() {
  // Virtual Tools 0, 1, 2, 3 = Filament 1, 2, 3, 4, etc.
  // Every virtual tool gets a pure filament
  /*LOOP_L_N(t, MIXING_VIRTUAL_TOOLS && t < MIXING_STEPPERS)
    MIXER_STEPPER_LOOP(i)
      color[t][i] = (t == i) ? COLOR_A_MASK : 0;*/
  color[0][0] = COLOR_A_MASK; // FORCED VALUE FOR E0
  color[1][1] = COLOR_A_MASK; // FORCED VALUE FOR E1
[...] 
```
I know that this isn't a really fix, but it works :)

**Comment by thinkyhead** [comment]
Please test the [`bugfix-2.0.x` branch](https://github.com/MarlinFirmware/Marlin/archive/bugfix-2.0.x.zip) to see where it stands. If the problem has been resolved then we can close this issue. If the issue isn't resolved yet, then we should investigate further.

**Comment by Exalik** [comment]
This issue has been recently solved in the bugfix-2.0.x
